cluster/options.go

```diff
@@ -4,8 +4,16 @@ import "crypto/tls"
 
 // Options is exported
 type Options struct {
-	TLSConfig       *tls.Config
+	TLS             *TLSConfig
 	OvercommitRatio float64
 	Discovery       string
 	Heartbeat       uint64
 }
+
+// TLSConfig is exported
+type TLSConfig struct {
```

Comment:
do we still need the `type TLSConfig struct` then ? I think we can just use `Config *tls.Config`